src/complete.rs

```diff
@@ -1695,7 +1695,7 @@ impl<'ctx> Completer<'ctx> {
             };
 
             let mut desc = WString::new();
-            if self.flags.descriptions && self.flags.autosuggestion {
+            if self.flags.descriptions && !self.flags.autosuggestion {
```

Comment:
Wow - `git blame` shows this regressed in 17bd7d0e40 (Switch completion_request_options_t from a list of flags to a struct, 2022-06-07).
Crazy that now one noticed.
Will cherry-pick this one to the release branch.